must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -207,9 +207,15 @@ for ns in $namespaces; do
 
     crash_collection(){
         oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        wait_period=0
         while [ "$(oc get pods -n openshift-storage | grep "${node//./}-debug" | awk '{print $2}')" != "1/1" ] ; do
-            sleep 3
-            echo "waiting for the debug pod to be in ready state"
+            wait_period=$wait_period+3
+            if [ $wait_period -gt 300 ];then
+                break
+            else
+                sleep 3
+                echo "waiting for the debug pod to be in ready state"
+            fi
         done
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/crash/ "${CRASH_OUTPUT_DIR}"
```

Comment:
If the debug Pod never comes up, we shouldn't try to run the rsync command. Maybe track another variable `debug_pod_timeout=false` and set it to `true` if we time out. Alternatively, you could restructure the while loop as follows:

```bash
debug_pod_ready=false
for i in {0..300..3); do
  if <check_debug_pod>; then
    debug_pod_ready=true
    break
  else
    sleep 3
  fi
done

if debug_pod_ready; then
  <oc_rsync>
fi
```

Reply:
this has been done

Reply:
yes